app/components/toggle-button.tsx

```
@@ -0,0 +1,32 @@
+    let showIcon = props.showIcon;
+    if (props.icon === undefined || props.icon === '') {
+        showIcon = false;
+    }
+    else if (props.showIcon === undefined) {
+        showIcon = false;
+    }
+    if (props.selected === undefined) {
+        props.selected = false;
+    }
+    return(
+        <div className={`toggle-btn flex items-center focus:ring ${props.selected ? 'selected' : ''}`} role='button' onClick={props.onClick}>
```

Biome

[error] The HTML element with the interactive role 'button' is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like 'button'.
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)